components/globe.tsx

```diff
@@ -1,6 +1,4 @@
-import React from 'react';
-
-const Globe = (props) => (
+const Globe: React.FC = (props) => (
   <svg width="46" height="46" viewBox="0 0 46 46" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
```

Comment:
is there a reason this isn't being `import`ed and plopped in via webpack?

(that way the SVG can be binary in git and not give xboxhueg diffs)

Reply:
We typically put SVGs in components so that props can be passed in. I don't feel strongly

Reply:
Never done that, but that makes sense. :+1: Just makes the diffs a bit of a chonk.